language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/BlockNodeContextProvider.java

```diff
@@ -194,6 +197,24 @@ protected List<LSCompletionItem> getStatementCompletionItems(BallerinaCompletion
         return completionItems;
     }
 
+    @Override
+    protected List<LSCompletionItem> getCompletionItemsOnQualifiers(Node node, BallerinaCompletionContext context) {
+        List<LSCompletionItem> completionItems = new ArrayList<>(super.getCompletionItemsOnQualifiers(node, context));
+        if (node.kind() == SyntaxKind.MODULE_VAR_DECL) {
```

Comment:
Is this a possible scenario inside a block node context?

Reply:
Yes, with the new parser change it's possible. See the issue #37039 and parser PR: #36973. This happens only after `isolated f<cursor>`

Reply:
Wait. Just got what you said. Seems like you are correct.

Reply:
Done!